### PR TITLE
Add error handling to signature maker.

### PR DESCRIPTION
**Pull request by surbhi-gulati**
Fixes [#18](https://github.com/HackBeanpot/internal-tools/issues/18).

- If the name, title, phone, or email field is empty, then there is an error for that field
- Clicking generate signature does not generate a signature if there are errors 

Images:
unfilled
incomplete
filledout

### CODE REVIEW

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
This looks great!! Just a few comments: 
- Can we make the inputs wider? Similar to the width from before? We have enough space and it'd be nice to see the whole @hackbeanpot.com placeholder
Screen Shot 2022-06-27 at 10 08 49 AM
- The deploy is failing with the following error which prevents us from merging this
Screen Shot 2022-06-27 at 10 09 52 AM
 
If you search the error on google there's several solutions so I recommend trying these out https://stackoverflow.com/questions/41462729/typescript-react-could-not-find-a-declaration-file-for-module-react-material

**Review by Dhivas-Sugumar — APPROVED** [review]
Great PR, looks good to me! Just add Judy's requested changes and should be gtg!

**Comment by surbhi-gulati** [comment]
Fixed! PTAL @jzhzhang66

File: pages/sigmaker.tsx [hunk 1]
```diff
@@ -150,40 +159,31 @@ const Sigmaker: NextPage = () => {
...
-                  setSignatureData(formData)
-                  setFormData({
-                    fullName: '',
-                    title: '',
-                    phone: '',
-                    email: ''
-                  })
-                }}
+                type="submit"
+                sx={{width:220}}
```
Comment (jzhzhang66) [inline pages/sigmaker.tsx:176]: We don't want to use inline styling - to make this button wider you can use width="medium"

File: pages/sigmaker.tsx [hunk 2]
```diff
@@ -59,15 +58,25 @@ const Sigmaker: NextPage = () => {
...
+    setFormData({
+      fullName: '',
+      title: '',
+      phone: '',
+      email: ''
+    })
+  }
+
+  const createValidatedInputField = (name: string, value: string, label: string) => (
+    <TextValidator
```
Comment (jzhzhang66) [inline pages/sigmaker.tsx:72]: To increase the input width, you'd want to create something like StyledTextValidator in sigmaker.styles.ts and set the width there. Then use StyledTextValidator here instead. I think a width of 275 or 280 would be pretty similar to what we had before! You can look at the way other components are styled for reference:)

File: pages/sigmaker.tsx [hunk 3]
```diff
@@ -150,40 +159,31 @@ const Sigmaker: NextPage = () => {
...
+                sx={{width:220}}
               >
                 Generate signature!
               </StyledButton>
-              <br />
-              <br />
-            </div>
+            </Stack>
+          </ValidatorForm>
+          <div>
```
Comment (jzhzhang66) [inline pages/sigmaker.tsx:182]: Somehow this button got re-added in - let's remove it (delete lines 182-186)

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
Nice job fixing the module! Left a few other comments - lmk if u have questions!!

**Review by jzhzhang66 — APPROVED** [review]
LGTM!!!

**Pull request merged**